### PR TITLE
Add a gaussian blur feature

### PR DESCRIPTION
**Pull request by yamilovs**
Hello guys! 
Can u check this PR?

PS. I'm sorry about another one (5 mins later) im new to github =)

### CODE REVIEW

File: Adapter/GD.php [hunk 1]
```diff
@@ -213,6 +213,56 @@ public function sepia()
     /**
      * {@inheritdoc}
```
Comment (Soullivaneuh) [inline Adapter/GD.php]: `gaussianBlur` does not exists on parent class.

This should be added on interface or have is proper php doc.

@Gregwar Adding to interface would be BC. A new major version should be released with an upgrade note.

**Comment by Soullivaneuh** [comment]
* Could you please add related tests?
* StyleCI is broken.

**Comment by yamilovs** [comment]
@Soullivaneuh hello!
I fixed string for StyleCI. But i think i cant add a test for Gaussian Blur. Its like smooth() or sepia() methods, and i dont know how i can test it) Sry)

**Comment by Soullivaneuh** [comment]
Should be here: https://github.com/Gregwar/Image/blob/master/tests/ImageTests.php

If you can't assert data, at least just run the method to see if it's running.

cc @Gregwar

**Comment by yamilovs** [comment]
@Soullivaneuh, ok i added a simple test for this filter;

**Comment by yamilovs** [comment]
and one more comment:
i have tried to run phpinit for this repo, but it failed with this message:

`Configuration read from /var/www/GregwarImage/phpunit.xml.dist`
`PHP Fatal error:  Class 'Gregwar\Cache\Cache' not found in /var/www/GregwarImage/Image.php on line 122`

is that ok?

**Comment by Gregwar** [comment]
@yamilovs you need to run composer install first

**Comment by yamilovs** [comment]
@Gregwar oh thanks for this comment. Its also work for me now)

**Comment by Soullivaneuh** [comment]
@Gregwar what do you think of https://github.com/Gregwar/Image/pull/104/files#r45178153?

**Comment by yamilovs** [comment]
Hello guys! any progress here?

**Comment by yamilovs** [comment]
ping @Gregwar 
I realy would like this feature in your library and in a symfony2 bundle =)

**Comment by williamrijksen** [comment]
+1 for this PR!

**Comment by cord** [comment]
+1 for this PR!

**Comment by Gregwar** [comment]
@Soullivaneuh is it ok for you?

**Comment by Gregwar** [comment]
Thanks for contributing

**Pull request merged**